### PR TITLE
Add template tag to determine if a user has access 

### PR DESCRIPTION
**Pull request by cklosowski**
#20

### CODE REVIEW

File: includes/template-tags.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	exit;
+}
+
+/**
+ * Determine if a user has permission to view the currently viewed URL
+ *
+ * Mainly for use in template files
+ * @return bool If the current user has permission to view the current URL
+ */
+function edd_cr_user_has_access() {
```
Comment (pippinsplugins) [inline includes/template-tags.php]: Can we have an option post and user ID paramter please?

Comment (cklosowski) [inline includes/template-tags.php]: See my note on the issue #20. I discussed this with Sean (someone who builds themes) and a `template tag` by default doesn't take any params, it simply is used in the themeing files in the loop. If we want a helper function, that's fine, but params is not a template tag.

Comment (cklosowski) [inline includes/template-tags.php]: The thing is, if someone wants to know if a user has access to a specific page, we have the function `edd_cr_user_can_access()`, which this template tag uses. The point of a template tag is to negate needing to pass in a bunch of data, in a template file, and get an answer without having to shuffle through parameters and logic.

Comment (pippinsplugins) [inline includes/template-tags.php]: That's why I said optional. If not passed in, it detects them. Nice to be flexible.

Most of WP core's template tags work the same way.

**Comment by cklosowski** [comment]
@pippinsplugins good now?

**Comment by pippinsplugins** [comment]
All good.

**Pull request merged**